packages/mask/src/plugins/Tips/SNSAdaptor/components/TipsRealmContent/index.tsx

```diff
@@ -89,11 +93,24 @@ export const TipsRealmContent: Plugin.InjectUI<Plugin.SNSAdaptor.TipsRealmOption
 
     if (slot === Plugin.SNSAdaptor.TipsSlot.MirrorMenu) {
         return (
-            <Guide>
-                <Stack display="inline-block" width="38px" height="38px" position="relative" top={2}>
-                    {button}
-                </Stack>
-            </Guide>
+            <PluginGuideProvider
+                value={{
+                    storageKey: EnhanceableSite.Mirror,
+                    totalStep: 1,
+                    storage: getStorage(),
```

Comment:
Why not create storage in `PluginGuideProvider`?

Reply:
I think the plugin can decides the storage provider.  The storage provider can be storage service or global variable. And plugin also should  keep and manage this value.

Reply:
The primary goal of a shared component is to reduce repeating working. No option is better than too many options.